source/settings/plugin/notify-status-plugin.rst

```diff
@@ -11,11 +11,11 @@ notify-status plugin
 Settings
 --------
 
-.. dovecot_plugin:setting:: notify_status_dict
+.. dovecot_plugin:setting:: notify_status/dict_driver
```

Comment:
Hmm. I wonder what would be the best way to describe these named filters. Maybe we need a whole new category like dovecot_plugin:setting_filter or something..

Reply:
Yeah, it took me a while to get what we would need.

For now i added  dovecot_plugin:setting_filter with :setting: (Required setting) and :filter: (Filter). I think these should link to the Required setting (dict_driver) in this example.

Reply:
I didn't get a chance to build and view this locally, but I see now that the issue here is the new 3.0 config syntax which introduces those "filter" blocks.  So if this compiles without error and displays, it's good for now.  There's no need to worry about additional linking (TODO), since we will try to handle that if/when we move to different documentation system.

Reply:
There are a couple of potential troubles:
 * In theory you can set a global dict_driver setting, and it would get used also for notify_status. So it's not exactly correct to say "The dict_driver to use. This MUST be set for the plugin to be active." in the filter since it could also be global. Of course, we should discourage using a global setting for this. So maybe this isn't really an issue after all.
 * We have oauth2 { .. } and auth_policy { .. } filters, which are mainly intended to allow overriding http settings for these lookups. They have no specific setting that are needed inside them. Can I use this new filter feature for these blocks without specifying a `:setting:`?